docs/resources/cbh_instance.md

```diff
@@ -0,0 +1,293 @@
+variable "security_group_id" {}
+variable "availability_zone" {}
+variable "region" {}
+
+resource "huaweicloud_cbh_instance" "test" {
+  flavor_id         = var.flavor_id
+  name              = "cbh_instance_test"
+  vpc_id            = var.vpc_id
+  availability_zone = var.availability_zone
+  region            = var.region
+  hx_password       = "test_123456",
+  bastion_type      = "OEM"
+  subnet_id         = var.subnet_id
+  security_group_id = var.security_group_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `flavor_id` - (Required, String, ForceNew) Specifies the product ID of the CBH server.
+
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the name of the CBH instance.
+
+  Changing this parameter will create a new resource.
+
+* `subnet_id` - (Required, String) Specifies the ID of a subnet.
+
+* `security_group_id` - (Required, String) Specifies the ID list of the security group.
+
+* `availability_zone` - (Required, String, ForceNew) Specifies the availability zone name.
+
+  Changing this parameter will create a new resource.
+
+* `hx_password` - (Required, String, ForceNew) Specifies the front end login password.
+
+  Changing this parameter will create a new resource.
+
+* `bastion_type` - (Required, String, ForceNew) Specifies the type of the bastion.
+
+  Changing this parameter will create a new resource.
+
+* `cloud_service_type` - (Required, String, ForceNew) Specifies the type of the CBH service.
+
+  Changing this parameter will create a new resource.
+
+* `charging_mode` - (Required, String, ForceNew) Specifies the charging mode of the read replica instance.
+  The options are as follows:
+  + **prePaid**: the yearly/monthly billing mode.
+
+  Changing this parameter will create a new resource.
+
+* `period_unit` - (Required, String, ForceNew) Specifies the charging period unit of the instance.
+  Valid values are *month* and *year*. This parameter is mandatory if `charging_mode` is set to *prePaid*.
+
+  Changing this parameter will create a new resource.
+
+* `period` - (Required, String, ForceNew) Specifies the charging period of the read replica instance.
+  If `period_unit` is set to **month**, the value ranges from 1 to 9.
+  If `period_unit` is set to **year**, the value ranges from 1 to 3.
+  This parameter is mandatory if `charging_mode` is set to **prePaid**.
+
+  Changing this parameter will create a new resource.
+
+* `auto_renew` - (Required, String, ForceNew) Specifies whether auto renew is enabled.
+  Valid values are "true" and "false". Defaults to **false**.
+
+  Changing this parameter will create a new resource.
+
+* `subscription_num` - (Required, Int, ForceNew) Specifies the number of the subscription.

```

Comment:
it seems the value of this parameter can only be `1`，how about just hide it

Reply:
done